src/state/user/reducer.ts

```diff
@@ -100,6 +102,7 @@ export const initialState: UserState = {
   timestamp: currentTimestamp(),
   audioPlay: true,
   isDark: false,
+  theme: null,
```

Comment:
Let's use `undefined` here. or else we have to add redux persist migration for any existing users

Reply:
Done